docs/source/changelog.rst

```diff
@@ -13,6 +13,7 @@ Changelog
         * Added error message when DateTimeIndex is a variable but not set as the time_index (:pr:`723`)
         * Fixed CumCount and other group-by transform primitives that take ID as input (:pr:`733`)
         * Fix progress bar undercounting (:pr:`743`)
+        * Fixed issue where some group-by features weren't created (:pr:`754`)
```

Comment:
Since this is fixing an unreleased bug we can add it as a second pr number on as the same line as pr 733